scripts/gha/test_simulator.py

```diff
@@ -59,12 +59,12 @@
 Device Information is stored in TEST_DEVICES in print_matrix_configuration.py
 Example:
 sdk id "system-images;android-29;google_apis;x86":
-  --android_sdk "system-images;android-29;google_apis;x86" --build_tools_version "28.0.3"
+  --android_sdk "system-images;android-29;google_apis;x86" --build_tools_version "29.0.2"
```

Comment:
Is there a need to update the version here since it is just an example?

Similarly to line 67

Reply:
No need, but I feel there is not harm too.